### DISCUSSION TITLE
Added an invalid character to a project file name and that file wont work

### DISCUSSION THREAD

**Discussion by qq735-personal** [LMMS/lmms#6891] [Support request]
it seems to have saved absolutely nothing, (the file doesnt have a filetype and is 0 bytes) and the weird part is it worked perfectly fine when I opened it through the recent projects menu.
the title of the project was "BATTLE THEME**:** OVERDRIVE
yes, i am aware i did a stupid. but is it in some sort of other location if it worked when i opened it through the recently opened tab? or is it gone?

**Comment by zonkmachine** [comment]
I just created a project titled:BATTLE THEME**:** OVERDRIVE
It saves and opens correctly so that doesn't seem to be the problem. If you can open the file from the recent projects menu fine, then save it once more under a different name.

**Comment by qq735-personal** [comment]
> I just created a project titled:BATTLE THEME**:** OVERDRIVE It saves and opens correctly so that doesn't seem to be the problem. If you can open the file from the recent projects menu fine, then save it once more under a different name.

i forgot to do that, so im past that point.
